packages/admin/cms-admin/src/index.ts

```diff
@@ -51,6 +51,7 @@ export { useDamConfig } from "./dam/config/useDamConfig";
 export { useCurrentDamFolder } from "./dam/CurrentDamFolderProvider";
 export { DamPage } from "./dam/DamPage";
 export { useDamFileUpload } from "./dam/DataGrid/fileUpload/useDamFileUpload";
+export type { DamRow } from "./dam/DataGrid/FolderDataGrid";
```

Comment:
imho that name is too generic (for a public export)

DamRowFragment? DamRowData?

Reply:
MAybe DamGridItem or DamGridRow?

Reply:
it's data structure (that contains just the data) vs. component (that renders it)

usually our exports are components, so I could think DamGridRow is also a component

Reply:
I could simply re-export the generated GraphQL type without a renaming, what do you think?

Reply:
The reason why you add it to the public api is, because it gets passed in renderAdditionalRowContent render prop.

final-form names the arguments passed to a render function eg `FieldRenderProps`, maybe you could use `AdditionalRowContentRenderProps` accordingly?